Update  5.1 ( Skipped $banID ) (#252)
NilPointer-Software/bdfd-wiki: src/bdscript/ban.md

# $ban
Bans the mentioned user.

## Syntax
```
$ban[(reason)]
```

## Parameters

> - `reason`: The reason for this action, which will be sent to the audit-log. _(Optional)_

Reviewer: ```suggestion
- `reason`: The reason for this action, which will be save in the audit log. Can be left empty.
```
